contract/src/main.rs

```diff
@@ -20,9 +20,15 @@ use alloc::{
     vec,
     vec::Vec,
 };
-use core::convert::TryInto;
+
+use constants::{ARG_EVENTS_MODE, EVENTS, EVENTS_MODE, EVENT_ID_TRACKER};
```

Comment:
```suggestion
use constants::{ARG_EVENTS_MODE, EVENTS, EVENTS_MODE};
```
as per previous comment

Reply:
Applied suggestion